packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -39,6 +39,24 @@ abstract class SliverChildDelegate {
   ) => null;
 
   bool shouldRebuild(@checked SliverChildDelegate oldDelegate);
+
+  @override
+  String toString() {
+    List<String> description = <String>[];
+    debugFillDescription(description);
+    return '$runtimeType#$hashCode(${description.join(", ")})';
+  }
+
+  @protected
+  void debugFillDescription(List<String> description) {
```

Comment:
That's not always what you want (for example, it would be quite reasonable for these subclasses to not call super here, since they override the "estimated" child count with a real child count).